src/app/view/components/modal/countScore.js

```diff
@@ -0,0 +1,11 @@
+function countScore({ questionsAnswered, questionsGuessed }) {
```

Comment:
I think you dont need that function here. @MonikaKrella created calculatePoints function for preparing the data.